### PR TITLE
Front-End Work Update

### PR DESCRIPTION
**Pull request by zbirmingham**
Cleaned up JavaScript and CSS, implemented partials template on all pages and created FAQ Form Validation

### CODE REVIEW

**Comment by zbirmingham** [comment]
@young @obraafo would you mind taking a look at this before I merge this

File: css/style_updated.css [hunk 1]
```diff
@@ -0,0 +1,859 @@
...
+
+.selectable_result {
+	font-family: 'Raleway', sans-serif;
+	font-weight: 300;
+	text-decoration: none;
+	color: #333333;
+}
+
+.selectable_result:hover {
+	background-color: #F15A24 !important;
```
Comment (young) [inline css/style_updated.css]: You shouldn't use '!important' unless you absolutely have to

File: index.html [hunk 2]
```diff
@@ -49,17 +50,14 @@
...
 			text-align: center;
-			background-color: #08516F;
 			width: 100px;
 			height: 4%;
 			display: table;
-			text-transform:lowercase;
+			text-transform: lowercase;
+			z-index: 2;
 			-webkit-backface-visibility: hidden;
 			-webkit-transform: translateZ(2);
```
Comment (young) [inline index.html]: Should just use transform: translateZ(2); without the prefix. It's pretty well [supported](http://caniuse.com/#search=translate).

File: index.html [hunk 3]
```diff
@@ -81,53 +78,8 @@
 			padding-right: 10px;
 			-webkit-transform: translateZ(2);
```
Comment (young) [inline index.html]: same

File: js/business-faq.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+	$('#header-search').load('./partials/search-bar.html');
+	// Load the nav bar
+	$('#nav-bar').load('./partials/nav-bar.html');
+	// Load 3 tabbed result boxes
+	$('#three-tabs').load('./partials/searchresults.html');
+}
+
+$(document).ready(function() {
+	loadPartials();
+	console.log(location.pathname);
```
Comment (young) [inline js/business-faq.js]: should remove console.log()

File: js/faq_form_validation.js [hunk 5]
```diff
@@ -0,0 +1,78 @@
+//Checks whether form is open and/or empty
+function check_form() {
```
Comment (young) [inline js/faq_form_validation.js]: Minor note: Javascript convention is to use camelcase for function names such as checkForm(). Python convention is to use underscores.

**Comment by zbirmingham** [comment]
Ok, I'll make the changes, my major concern is that I am deleting project dependent files.

**Comment by zbirmingham** [comment]
@young and @obraafo, let me know what you guys think. I made some changes, if you guys are good. I'll commit it to master.

**Comment by young** [comment]
Ship it

**Comment by zbirmingham** [comment]
@young thanks for taking a look

**Pull request merged**